server/tests/controller/moduleProcessor.test.ts

```diff
@@ -11,13 +11,23 @@ const TEST_DATA = {
     module_name: "School",
     description: "For school :3",
     completion_percent: 0,
-    email: "example@gmail.com"
+    email: "example@gmail.com",
+    coach_id: null
+}
+
+const TEST_DATA2 = {
```

Comment:
Don't exactly need to define another TEST_DATA, you can just simply add it unto the first one by changing the values from strings (or integers) into arrays. Just a little suggestion.